src/image.cpp

```diff
@@ -27,6 +27,7 @@ namespace librealsense
         case RS2_FORMAT_DISPARITY32: return 32;
         case RS2_FORMAT_XYZ32F: return 12 * 8;
         case RS2_FORMAT_YUYV:  return 16;
+        case RS2_FORMAT_M420:  return 12; // 16 pixels are represented with 24 bytes (16 of Y and 8 of Cr, Cb) - 24 / 16 * 8 = 12
```

Comment:
I don't understand the equation 24 / 16 * 8 = 12
Can you refine the comment to explain this calculation?

Reply:
24 bytes result to 16 pixels (16 Y and 8 U,V).
I see now it is explained in the comment.
In the code, we do shift by 3 (divide by 8), so I have multiplied by 8 here